udata/core/organization/forms.py

```diff
@@ -15,6 +15,24 @@
 )
 
 
+def org_bid_check(form, field):
+    if field.data:
+        # EID checks are country dependant. Following one is suitable for France.
+        if ORG_BID_FORMAT == 'siret':
+            siret_number = str(field.data)
+            # Checksum verification on only the SIREN part, the 9 first digits.
```

Comment:
I'm confused about this, we should be able to check for the entire SIRET, with La Poste as the only exception right?